baselines/Google Drive and Docs Minimum Viable Secure Configuration Baseline v0.2.md

```diff
@@ -98,10 +98,11 @@ Agencies SHALL disable making files and published web content visible to anyone
   - [T1537: Transfer Data to Cloud Account](https://attack.mitre.org/techniques/T1537/)
 
 #### GWS.DRIVEDOCS.1.6v0.2
-Agencies SHALL enable access checking for file sharing outside of Docs or Drive.
+Agencies SHOULD set access checking to recipients only.
```

Comment:
A little bit more work needs to be done here to bring the policy statement, the note, and the implementation instructions in-sync with each other.

Per the policy statement, there is only one acceptable option:
![image](https://github.com/cisagov/ScubaGoggles/assets/106177711/44bcf4c1-1a22-487e-b82d-712706ab533d)

Per the note, there are two acceptable options:
![image](https://github.com/cisagov/ScubaGoggles/assets/106177711/d4c804f4-8322-48d4-81a9-c6ef7a9585ba)

Finally, per the implementation, there is just one acceptable option:
![image](https://github.com/cisagov/ScubaGoggles/assets/106177711/509eec2d-d53f-4213-a0bc-f7812e69c8d6)

Reply:
our policy statement says it SHOULD be set to that option, but the note is telling them that another acceptable option is the other one and not to set it to the one that includes public. The way we have it is the way we intended it to be.